src/apps/views/AppsList/AppsList.tsx

```diff
@@ -0,0 +1,182 @@
+import useListSettings from "@saleor/hooks/useListSettings";
+import useNavigator from "@saleor/hooks/useNavigator";
+import useNotifier from "@saleor/hooks/useNotifier";
+import usePaginator, {
+  createPaginationState
+} from "@saleor/hooks/usePaginator";
+import { commonMessages } from "@saleor/intl";
+import { ListViews } from "@saleor/types";
+import createDialogActionHandlers from "@saleor/utils/handlers/dialogActionHandlers";
+import React, { useEffect, useMemo } from "react";
+import { useIntl } from "react-intl";
+import { RouteComponentProps } from "react-router-dom";
+
+import { AppSortField, OrderDirection } from "../../../types/globalTypes";
+import AppsListPage from "../../components/AppsListPage";
+import {
+  useAppDelete,
+  useAppInstallMutation,
+  useAppManifestFetchMutation,
+  useAppRetryInstallMutation
+} from "../../mutations";
+import {
+  useAppsInProgressListQuery,
+  useInstalledAppsListQuery
+} from "../../queries";
+import { AppDelete } from "../../types/AppDelete";
+import {
+  AppListUrlDialog,
+  AppListUrlQueryParams,
+  appsListUrl,
+  appUrl
+} from "../../urls";
+
+const MANIFEST_ATTR = "manifestUrl";
```

Comment:
It should be placed in `src/apps/urls.ts` since it's a part of a `params` object